NuGet.config

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <add key="darc-pub-dotnet-core-setup-7d57652" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-7d57652f/nuget/v3/index.json" />
     <add key="darc-pub-dotnet-corefx-4ac4c03" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-4ac4c036/nuget/v3/index.json" />
     <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
+    <clear />
```

Comment:
@mmitche I'm a bit surprised this PR validated properly given I've seen mention of moving the `<clear />` arround being a bug. Did this update succeed only because Extensions has already moved on from GA packages?

Reply:
Because all the packages (stable) that were in those feeds have been published to nuget.org.

Reply:
Is the dependency update code smart enough to move the `<clear />` back up when it next changes the feeds in the comments?

Reply:
Yes